collectors/log.go

```diff
@@ -17,21 +17,6 @@ var (
 
 	// Logger for lndmon's main process.
 	Logger = backendLog.Logger("LNDMON")
-
```

Comment:
?

As they were never used?

Reply:
These loggers were only used to log errors when we couldn't query lnd, which is now replaced by bubbling the error up through `errChan`. I don't _love_ deleting them, because we could potentially re-add them in future with different prefixes, which would be inconsistent for end users, but the linter is complaining. Could add some placeholder log for each collector so we don't need to delete?